doc/install.rst

```diff
@@ -44,39 +41,44 @@ To use PySurfer, you will need to have the following Python packages:
 * mayavi_
 * matplotlib_
 
-Some functions also make use of the Python Imaging Library (PIL_), although
-it's not mandatory.
+Some input/output functions also make use of the Python Imaging Library (PIL_)
+and ``imageio``, although they are not mandatory.
 
 An easy option to set up this environment is the Anaconda_ distribution, which
 is free and ships with many of the required packages. If you use Anaconda,
-you'll need to install Mayavi separately. This can be done using the ``conda``
-command::
+you'll need to install Mayavi separately. This should be done using ``conda``::
 
     conda install mayavi
```

Comment:
I doubt this actually works beyond 2.7. @larsoner we should use the install procedure as explained for MNE.